colossalai/inference/server/README.md

```diff
@@ -1,27 +1,33 @@
 # Online Service
-Colossal-Inference supports fast-api based online service. Simple completion and chat are both supported. Follow the commands below and
-you can simply construct a server with both completion and chat functionalities. For now we only support `Llama` model, we will fullfill
-the blank quickly.
+Colossal-Inference supports fast-api based online service. Simple completion and chat are both supported. Follow the commands below and you can simply construct a server with both completion and chat functionalities. For now we support `Llama`,`Glide_Llama` and `baichuan` model, we will fullfill the blank quickly.
```

Comment:
Please migrate this README to an appropriate section in colossalai/inference/README.md and remove this file.

Reply:
done